web-common/src/features/models/selectors.ts

```diff
@@ -31,6 +37,26 @@ export function useModel(instanceId: string, name: string) {
   return useResource(instanceId, name, ResourceKind.Model);
 }
 
+export function useAllModelColumns(
+  queryClient: QueryClient,
+  instanceId: string
+): Readable<Array<TableColumnsWithName>> {
+  return derived([useModels(instanceId)], ([allModels], set) => {
+    if (!allModels.data?.length) {
+      set([]);
+      return;
+    }
+
+    derived(
+      allModels.data.map((r) =>
+        createTableColumnsWithName(queryClient, instanceId, r.meta.name.name)
+      ),
+      (sourceColumnResponses) =>
```

Comment:
Ah, this tipped me off to where you got this code :). Nice approach to just mirror what we have for sources. But: `sourceColumnResponses` -> `modelColumnResponses`

Reply:
Renamed